_posts/2023-01-04-trino-2022-the-rabbit-reflects.md

```diff
@@ -0,0 +1,581 @@
+integration. This also included a range of bug fixes.
+
+### Airflow integration
+
+The long-awaited [Trino/Airflow integration
+](https://airflow.apache.org/docs/apache-airflow-providers-trino/stable/index.html)
+landed this year. This paired well with the new task-retry and fault-tolerant
+execution features. There was a small tutorial on [how to use Trino and Airflow
+]({% post_url 2022-07-13-how-to-use-airflow-to-schedule-trino-jobs %})
+from Willie Valdez that details how to get started. To learn more about the full
+capabilities of pairing Trino’s few fault-tolerant execution mode with Airflow,
+check out [Philippe Gagnon’s talk at this year’s Trino Summit
+](https://www.youtube.com/watch?v=xKDN7RUJ5i4).
+
+### Metabase driver
+
+A lot of folks in the community were asking for a [Trino/Metabase driver
+](https://github.com/metabase/metabase/issues/17532) after Trino updated its
+name. This was a large blocker for anyone who wanted to move over to Trino but
+Metabase was the primary BI tool used. Through some collaboration with the
+Metabase and Starburst folks, the [metabase-drive
+](https://github.com/starburstdata/metabase-driver) for Trino/Starburst made its
+way in earlier this year, and we saw quite a few folks migrate over from Presto
+with this new open source driver.
+
+## 2023 Roadmap
+
+The upcoming roadmap was [covered in detail](https://youtu.be/mUq_h3oArp4?t=799)
+by Martin at Trino Summit. To avoid too extending this blog even further, we’ll
+leave you with the featured project that covers many aspects of the Trino core
+engine.
+
+### Project Hummingbird
+
+[Project Hummingbird](https://github.com/trinodb/trino/issues/14237) aims to
+improve Trino's columnar and vectorized evaluation engine. Every year we report
+on many incremental performance improvements. These improvements are typically
+small in isolation but have a large aggregate impact as they start to compound
+and overlap with more common workloads being run in production. This incremental
+approach is the real key to improving query engine performance, but there is
+still room for further optimization. It’s time to take Trino to the next level
+by bringing improvements in areas such as filter, projection, aggregation and
+join evaluation, as well as any other potential improvements in areas. If you
+want to get involved with this exciting project, or to learn about the latest
+innovations as they are being discussed, joint the #project-hummingbird in
+[the Trino Slack workspace]({{site.url}}/slack.html).
+
+## Conclusion
+
+2022 was by far the busiest this bunny has been. While we’re all kind of waiting
+for the moment where the trajectory of things going on would start to taper off,
+it’s only growing as we attract more contributors. We believe that this trend
+will continue in 2023 as we begin to put more process in place around managing
+pull requests. We are also in the process of building out a library of Trino
```

Comment:
Lets not talk about the user group, presentation and other stuff just yet .. lets announce when we have more in a separate blog post and create a call out for calendar entries, speakers and more